docs/source/data-model.rst

```diff
@@ -274,7 +274,23 @@ Formal schema:
 Event contents can also be represented in "paged" form, where multiple
 rows are contained in one structure for efficient transport and vectorized
 computation. The representations contain equivalent information: an EventPage
-can always be losslessly transformed into an Event and vice versa.
+can always be losslessly transformed into an Event and vice versa. Here is the
```

Comment:
... can always be transformed, without loss, into ...

Reply:
Done. Always good to avoid needless jargon when we  can.